container-images/tcib/base/os/tempest/run_tempest.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+
+HOMEDIR=/var/lib/tempest
+TEMPEST_DIR=$HOMEDIR/openshift
+
+pushd $HOMEDIR
+
+export OS_CLOUD=default
+
+TEMPEST_PATH=$HOMEDIR/
+if [ ! -z ${USE_EXTERNAL_FILES} ]; then
+    TEMPEST_PATH=$HOMEDIR/external_files/
+    mkdir -p $HOME/.config/openstack
+    cp ${TEMPEST_PATH}clouds.yaml $HOME/.config/openstack/clouds.yaml
+fi
+
+tempest init openshift
+
+pushd $TEMPEST_DIR
+
+discover-tempest-config --os-cloud $OS_CLOUD --debug --create identity.v3_endpoint_type public
+
+if [ ! -f ${TEMPEST_PATH}include.txt ]; then
+    echo "tempest.api.identity.v3" > ${TEMPEST_PATH}include.txt
+fi
+if [ ! -f ${TEMPEST_PATH}/exclude.txt ]; then
```

Comment:
the '/' is redundant and not sure if it will cause problems since 

TEMPEST_PATH=$HOMEDIR/

you didn't include it above for the include.txt

sorry I didn't notice on last pass